lms/djangoapps/api_manager/courses/tests.py

```diff
@@ -418,11 +418,11 @@ def test_courses_tree_get(self):
         chapter = response.data['content'][0]
         self.assertEqual(chapter['category'], 'chapter')
         self.assertEqual(chapter['name'], 'Overview')
-        self.assertEqual(len(chapter['children']), 6)
+        self.assertEqual(len(chapter['children']), 5)
 
         sequence = chapter['children'][0]
-        self.assertEqual(sequence['category'], 'videosequence')
-        self.assertEqual(sequence['name'], 'Video_Sequence')
+        self.assertEqual(sequence['category'], 'sequential')
```

Comment:
As with the above comment, please add a note here describing why the sequential is the first item in the list now (versus the previous videosequence) -- also it would be better to not depend on the array ordering -- instead we should search the array for what we are interested in asserting to ensure it is there.  Unless of course the sequential absolutely MUST be in the first position of the array -- if that's the case then we should note this with a comment.